Detections/SecurityEvent/UserAccountCreatedDeleted_10m.yaml

```diff
@@ -12,7 +12,7 @@ requiredDataConnectors:
     dataTypes:
       - SecurityEvent
 queryFrequency: 1d
-queryPeriod: 2d
+queryPeriod: 25h
```

Comment:
We can set this, doesn't really make any difference in the query because the timeframes are hard coded.  This is used if there is no timeframe specified which is generally preferred when possible.